Source/relay/TourGuide/Quests/Level 13.ash

```diff
@@ -153,7 +153,7 @@ boolean generateTowerFamiliarWeightMethod(string [int] how, string [int] immedia
     }
     else if (__misc_state["can eat just about anything"] || (__misc_state["can drink just about anything"] && __misc_state["VIP available"]))
     {
-        weight_modifiers.listAppend(TFWMInternalModifierMake("irradiated pet snacks (semi-rare, menagerie level 2)", false, false, true, 10.0));
+        weight_modifiers.listAppend(TFWMInternalModifierMake("irradiated pet snacks (lucky adventure, menagerie level 2)", false, false, true, 10.0));
```

Comment:
realistically this entire generateTowerFamiliarWeightMethod is a wildly heavyweight way to give advice on something that i believe does not exist at all in kol anymore, since i don't think a single class has the old tower familiar fight thing. but i guess this is fine to include even though it is super weird. part of me kind of wants to just nuke the file but i guess it isn't hurting anyone to include it.